packages/desktop-client/src/components/budget/BudgetTotals.js

```diff
@@ -80,7 +80,7 @@ const BudgetTotals = memo(function BudgetTotals({
                 }}
                 items={[
                   {
-                    name: 'toggleVisibility',
+                    name: 'toggle-visibility',
                     text: 'Toggle hidden',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏now that we use it in mobile app it would be better to improve the copy here. Previously the menu was closely connected to the categories list, but now it's in a different location. So users will be confused: "what is this going to toggle? why is nothing changing when I click it?".

```suggestion
                    text: 'Toggle hidden categories',
```